### PR TITLE
P0 input

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to me! 👍

File: css/components/badge.css [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 .badge-text{
   padding: 0.2rem 0.4rem;
-  width: auto;
+  width: fit-content;
   border-radius: 0.5rem;
```
Comment (deekshithmd) [inline css/components/badge.css]: use px for border-radius

File: css/components/card.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
+
+.card{
+  border-radius: 0.5rem;
+  border:none;
+}
+
+.card-vertical{
+  display:grid;
+  grid-gap: 0.3rem;
```
Comment (deekshithmd) [inline css/components/card.css]: for values less than 10px usin px will be good instead of rem

**Review by deekshithmd — APPROVED** [review]

File: css/components/card.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  padding:0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+} */
+
+/* .card-overlap-body{
+  grid-area: body;
+  padding:0.2rem;
+} */
```
Comment (parvez-rashid) [inline css/components/card.css]: Remove the comments if you don't have plans for it in refactoring.

**Review by parvez-rashid — APPROVED** [review]
Looks good to merge.

**Pull request merged**